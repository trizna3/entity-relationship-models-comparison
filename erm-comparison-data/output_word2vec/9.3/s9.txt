account;students-accommodated
contracts-contracts
payments-payments
buildings-addresses;buildings
rooms-floors;rooms
CONTRACT_11_ASSOCIATION:E-account-students
DECOMPOSE_ATTRIBUTE:S-address-addresses
CHANGE_CARDINALITY:S-addresses-buildings
CONTRACT_11_ASSOCIATION:S-addresses-buildings
CHANGE_CARDINALITY:S-floors-rooms
CONTRACT_11_ASSOCIATION:S-floors-rooms
