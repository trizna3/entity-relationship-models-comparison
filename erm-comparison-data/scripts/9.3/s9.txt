entity set addresses {
	"address"
};

entity set buildings {
	"name"
};

entity set rooms {
	"number"
	"capacity"
	"fee"
};

entity set floors {
	"number"
};

entity set contracts {
	"start"
	"end"
};

entity set accommodated {
	"name"
	"account"
};

entity set payments {
	"time"
};

association between 
{
	addresses MANY
	buildings ONE
}{};

association between 
{
	buildings MANY 
	floors MANY
}{};

association between 
{
	floors MANY 
	rooms ONE
}{};

association between 
{
	rooms MANY
	contracts ONE
}{};

association between 
{
	contracts MANY 
	accommodated ONE
}{};

association between 
{
	contracts ONE 
	payments MANY
}{};