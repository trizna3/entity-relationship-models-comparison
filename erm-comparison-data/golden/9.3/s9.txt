buildings-buildings
rooms-rooms
contracts-contracts
payments-payments
students;account-accommodated
CONTRACT_11_ASSOCIATION:E-students-account